### ISSUE TITLE
unknown option `long'

### ISSUE THREAD

**Issue by horejsek** [schreifels/git-num#6]
Probably it works only with some versions of Git and there is no information about which one. 

```
$ git num
error: unknown option `long'
usage: git status [options] [--] <filepattern>...
```

I am using Debian Wheezy with this Git version:

```
$ git --version
git version 1.7.10.4
```

**Comment by schreifels** [comment]
I can reproduce this. It looks like that particular option is the only compatibility problem in Git 1.7.x, so I will fix this. I will also add explicit version requirements to the README for both Git and Ruby.

Thanks for the report!

**Comment by horejsek** [comment]
Thanks. :-)

**Comment by schreifels** [comment]
This is fixed in v2.0.1. Enjoy!

**Comment by horejsek** [comment]
Works like a charm, thank you. :-)
